ledger/pool/pool_fork_checker.go

```diff
@@ -190,7 +190,7 @@ func (pl *pool) checkIrreversiblePrinciple(keyPoint *snapshotPoolBlock) error {
 }
 
 func (pl *pool) updateIrreversibleBlock() error {
-	nodeCnt := pl.cs.SBPReader().GetNodeCount()
+	nodeCnt := pl.nodeCnt
```

Comment:
Not sure, but with this change the `nodeCnt` will be static. Should we pass the `SBPReader` instead of `periodTimeIndex` and `nodeCnt` when initializing the pool with `Init`?